packages/main/src/plugin/container-registry.ts

```diff
@@ -993,8 +1003,8 @@ export class ContainerProviderRegistry {
       if (engine) {
         await this.attachToContainer(engine, container);
       }
-
-      return container.start();
+      // see https://github.com/containers/podman-desktop/issues/4363
```

Comment:
IMHO we can remove the link to the podman desktop issue, we only reference issues in the commit message / PR description

each time we do changes in the codebase we don't add backlinks to the repository
we can track the links by looking at the git changes and get back to the commit